shardingsphere-infra/shardingsphere-infra-context/src/test/java/org/apache/shardingsphere/infra/context/refresher/type/CreateTableStatementSchemaRefresherTest.java

```diff
@@ -0,0 +1,92 @@
+import org.apache.shardingsphere.sql.parser.sql.dialect.statement.postgresql.ddl.PostgreSQLCreateTableStatement;
+import org.apache.shardingsphere.sql.parser.sql.dialect.statement.sql92.ddl.SQL92CreateTableStatement;
+import org.apache.shardingsphere.sql.parser.sql.dialect.statement.sqlserver.ddl.SQLServerCreateTableStatement;
+import org.junit.Test;
+
+import javax.sql.DataSource;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Collection;
+import java.util.Map;
+
+import static org.junit.Assert.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.RETURNS_DEEP_STUBS;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public final class CreateTableStatementSchemaRefresherTest {
+    
+    @Test
+    public void refreshForMySQL() throws SQLException {
+        MySQLCreateTableStatement createTableStatement = new MySQLCreateTableStatement();
+        createTableStatement.setContainsNotExistClause(false);
+        refresh(createTableStatement);
+    }
+    
+    @Test
+    public void refreshForOracle() throws SQLException {
+        OracleCreateTableStatement createTableStatement = new OracleCreateTableStatement();
+        refresh(createTableStatement);
+    }
+    
+    @Test
+    public void refreshForPostgreSQL() throws SQLException {
+        PostgreSQLCreateTableStatement createTableStatement = new PostgreSQLCreateTableStatement();
+        createTableStatement.setContainsNotExistClause(false);
+        refresh(createTableStatement);
+    }
+    
+    @Test
+    public void refreshForSQL92() throws SQLException {
+        SQL92CreateTableStatement createTableStatement = new SQL92CreateTableStatement();
+        refresh(createTableStatement);
+    }
+    
+    @Test
+    public void refreshForSQLServer() throws SQLException {
+        SQLServerCreateTableStatement createTableStatement = new SQLServerCreateTableStatement();
+        refresh(createTableStatement);
+    }
+    
+    private void refresh(final ShardingSphereMetaData schemaMetaData, final FederationSchemaMetaData schema, final Map<String, OptimizerPlannerContext> optimizerPlanners, 
+                         final Collection<String> logicDataSourceNames, final CreateTableStatement sqlStatement, final ConfigurationProperties props) throws SQLException {
+        sqlStatement.setTable(new SimpleTableSegment(new TableNameSegment(1, 3, new IdentifierValue("t_order_0"))));
+        DataSource dataSource = mock(DataSource.class, RETURNS_DEEP_STUBS);
+        when(dataSource.getConnection().getMetaData().getTables(any(), any(), any(), any())).thenReturn(mock(ResultSet.class));
+        //ShardingSphereSchema schema = ShardingSphereSchemaBuildUtil.buildSchema();
+        //ShardingSphereMetaData metaData = new ShardingSphereMetaData("", mock(ShardingSphereResource.class), mock(ShardingSphereRuleMetaData.class), schema);
```

Comment:
Please do not leave commented codes.

Reply:
Ok

Reply:
Hi @terrymanu , why the [OracleCreateTableStatement.java](https://github.com/apache/shardingsphere/blob/master/shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/dialect/statement/oracle/ddl/OracleCreateTableStatement.java) is empty? I mean it does not implements anything.